dogecoin/dogecoin
# Update static seeds

Updates static seeds for 1.14.7. 

Entries were discovered through network peer exchange, no DNS server was used to reduce potential bias.

**Conditions applied to discovered peers:**

- must have an up-to-date protocol version and perform handshake properly (because otherwise they will be disconnected before `getaddr`)
- must have up-to-date `addr` processing logic and respond to `getaddr` (because that's all we need them to do)
- any evidence of non-standard behavior or being at max capacity lowers inclusion priority

I've limited mainnet to 250 seed nodes.

-----

**To test (on command line linux):**

This facility provides your node with peers regardless of the dns servers being reachable.

1. delete `~/.dogecoin/peers.dat` (`~/.dogecoin/testnet3/peers.dat` for testnet)
2. make sure that there is no `addnode` in `dogecoin.conf`
3. run `dogecoind` with `-dnsseed=0 -onlynet=ipv4` to test ipv4 seeds; make sure it connects to peers
4. run `dogecoind` with `-dnsseed=0 -onlynet=ipv6` to test ipv6 seeds; make sure it connects to peers
